src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsServiceTest.java

```diff
@@ -229,112 +227,6 @@ public void strictBandwidthUpdateTest() throws FlowNotFoundException, InvalidFlo
         assertTrue(updatedFlow.isStrictBandwidth());
     }
 
-    @Test
-    public void updateVlanStatisticsTest() throws FlowNotFoundException, InvalidFlowException {
-        String testFlowId = "flow_id";
-        Set<Integer> originalVlanStatistics = new HashSet<>();
-        originalVlanStatistics.add(11);
-
-        Flow flow = new TestFlowBuilder()
-                .flowId(testFlowId)
-                .srcSwitch(switchA)
-                .destSwitch(switchB)
-                .vlanStatistics(originalVlanStatistics)
-                .build();
-        flowRepository.add(flow);
-
-        Set<Integer> expectedVlanStatistics = new HashSet<>();
-        expectedVlanStatistics.add(31);
-
-        FlowPatch receivedFlow = FlowPatch.builder()
-                .flowId(testFlowId)
-                .vlanStatistics(expectedVlanStatistics)
-                .build();
-
-        Flow updatedFlow = flowOperationsService.updateFlow(new FlowCarrierImpl(), receivedFlow);
-
-        assertThat(updatedFlow.getVlanStatistics(), containsInAnyOrder(expectedVlanStatistics.toArray()));
```

Comment:
You should not delete this test, but change it. This test must check that `updateFlow` would NOT change the vlan stats in the flow

Reply:
Updated.